src/zenml/integrations/gcp/model_deployers/vertex_model_deployer.py

```diff
@@ -0,0 +1,270 @@
+                    continue
+                return False, (
+                    f"The '{stack_comp.name}' {stack_comp.type.value} is a "
+                    f"local stack component. The Vertex AI Pipelines "
+                    f"orchestrator requires that all the components in the "
+                    f"stack used to execute the pipeline have to be not local, "
+                    f"because there is no way for Vertex to connect to your "
+                    f"local machine. You should use a flavor of "
+                    f"{stack_comp.type.value} other than '"
+                    f"{stack_comp.flavor}'."
+                )
+
+            return True, ""
+
+        return StackValidator(
+            required_components={
+                StackComponentType.MODEL_REGISTRY,
+            },
+            custom_validation_function=_validate_stack_requirements,
+        )
+
+    def _create_deployment_service(
+        self, id: UUID, timeout: int, config: VertexAIDeploymentConfig
+    ) -> VertexDeploymentService:
+        """Creates a new VertexAIDeploymentService.
+
+        Args:
+            id: the UUID of the model to be deployed with Vertex model deployer.
+            timeout: the timeout in seconds to wait for the Vertex inference endpoint
+                to be provisioned and successfully started or updated.
+            config: the configuration of the model to be deployed with Vertex model deployer.
+
+        Returns:
+            The VertexModelDeployerConfig object that can be used to interact
+            with the Vertex inference endpoint.
+        """
+        # create a new service for the new model
+        service = VertexDeploymentService(uuid=id, config=config)
+        logger.info(
+            f"Creating an artifact {VERTEX_SERVICE_ARTIFACT} with service instance attached as metadata."
+            " If there's an active pipeline and/or model this artifact will be associated with it."
+        )
+        service.start(timeout=timeout)
+        return service
+
+    def perform_deploy_model(
+        self,
+        id: UUID,
+        config: ServiceConfig,
+        timeout: int = DEFAULT_DEPLOYMENT_START_STOP_TIMEOUT,
+    ) -> BaseService:
+        """Deploy a model to Vertex AI.
+
+        Args:
+            id: the UUID of the service to be created.
+            config: the configuration of the model to be deployed.
+            timeout: the timeout for the deployment operation.
+
+        Returns:
+            The ZenML Vertex AI deployment service object.
+        """
+        with track_handler(AnalyticsEvent.MODEL_DEPLOYED) as analytics_handler:
+            config = cast(VertexAIDeploymentConfig, config)
+            service = self._create_deployment_service(
+                id=id, config=config, timeout=timeout
+            )
+            logger.info(
+                f"Creating a new Vertex AI deployment service: {service}"
+            )
+            service.start(timeout=timeout)
```

Comment:
`_create_deployment_service` already starts the server, can we remove one of the two calls?